tests/phpunit/tests/widgets.php

```diff
@@ -1364,4 +1364,24 @@ public function test_sidebar_guessing_one_menu_per_sidebar() {
 		);
 		$this->assertSameSetsWithIndex( $expected_sidebars, $new_next_theme_sidebars );
 	}
+
+	/**
+	 * @ticket 57531
+	 */
+	public function test_stores_classic_sidebars_when_switching_to_block_theme() {
+		global $wp_registered_sidebars;
+
+		$this->register_sidebars( array( 'sidebar-1', 'sidebar-2' ) );
+
+		switch_theme( 'block-theme' );
+
+		unregister_sidebar( 'sidebar-1' );
+		unregister_sidebar( 'sidebar-2' );
+
+		// Register stored classic sidebars for block themes..
```

Comment:
```suggestion
		// Register stored classic sidebars for block themes.
```

Remove extra full stop.